src/NaturalLanguage/NLLanguageRecognizer.cs

```diff
@@ -54,14 +57,20 @@ public static NLLanguage GetDominantLanguage (string @string)
 			}
 			set {
 				var i = 0;
+				var skipCount = 0;
 				var nsKeys = new NSString[value.Keys.Count];
 				var nsValues = new NSNumber[value.Keys.Count];
 				foreach (var item in value) {
-					nsKeys[i] = NLLanguageExtensions.GetConstant (item.Key);
+					var constant = NLLanguageExtensions.GetConstant (item.Key);
+					if (constant is null) {
+						skipCount++;
+						continue;
+					}
+					nsKeys [i] = constant;
 					nsValues[i] = new NSNumber (item.Value);
 					i++;
 				}
-				NativeLanguageHints = NSDictionary<NSString, NSNumber>.FromObjectsAndKeys (nsValues, nsKeys, nsKeys.Length);
+				NativeLanguageHints = NSDictionary<NSString, NSNumber>.FromObjectsAndKeys (nsValues, nsKeys, nsKeys.Length - skipCount);
```

Comment:
The above changes were to account for `NLLanguageExtensions.GetConstant (item.Key)` possibly being null, but not sure if all this was neccessary

Reply:
I think I would just force the GetConstant return value:

```cs
nsKeys[i] = NLLanguageExtensions.GetConstant (item.Key)!;
```

to avoid changing behavior.